src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const styleObject = {};
+
+  stylesArray.forEach((style) => {
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy of the array before using `forEach`. The original array is not mutated by `forEach`, so you can safely use it directly.